static/index.html

```diff
@@ -42,6 +42,7 @@
         </div>
 
         <p id="timer"></p>
+        <p id="init_info_1"></p>
```

Comment:
init_info_1 is such a not explanatory name. I think you should name it somehow different to be easier to understand what is it

Reply:
What name would be better for this element?
I can't realise that, because i think it's okey

Reply:
Well in no world (almost) is naming a variable `something_1` a good idea. As a person who tries to understand what is happening, I cannot understand why is it created, what is stands for.
So a better name would be `processing_status` as `init_info` was originally just a console from server that printed out initialization information and current steps